remoteappmanager/db/orm.py

```diff
@@ -129,6 +130,54 @@ def reset(self):
         Base.metadata.create_all(self.engine)
 
 
+class AccountingInterface(ABCAccounting):
+
+    def __init__(self, url, **kwargs):
+        self.db = Database(url, **kwargs)
+
+        # We keep the same session for all transactions
+        # so that the session remains on one thread
+        self.session = self.db.create_session()
```

Comment:
I had to do this otherwise when calling `get_apps_for_user` from any coroutine of the HomeHandler, sqlalchemy complains about accessing the session from a different thread.